README.md

```diff
@@ -21,71 +21,30 @@
 ember install ember-frost-modal
 ```
 
-## API
-| Attribute | Type | Value | Description |
-| --------- | ---- | ----- | ----------- |
-| `modalName` | `string` | <name> | Optional name for the modal |
-| `modalClass` | `string` | <class-name> | Optional class to add custom styles to the modal |
-| `closeOnClick` | `boolean` | `default = true` | `Modal closes when the Confirm button is clicked` |
-| | | `false` | `Modal does not automatically close when the Confirm button is clicked` |
+## API and Examples
+Detailed API and example usage can be found in the sample application in `tests/dummy`, which is also running at http://ciena-frost.github.io/ember-frost-modal
 
-### Ember-remodal
+### Ember-elsewhere
 
-This addon uses the [ember-remodal](http://sethbrasile.github.io/ember-remodal/) service as a base for the modal behavior.
+This addon uses the [ember-elsewhere](https://github.com/ef4/ember-elsewhere) to manage the modals, to lift them outside the normal route hiearchy.
 
-### Slots API
-
-Using [ember-block-slots](https://github.com/ciena-blueplanet/ember-block-slots), this generic modal dialog can be wrapped inside other more complex modal components. This basic modal will provide a header, scrollable content area and footer with actions. The required slots are `target` and `footer`, to provide a way to launch and close the modal dialog respectively.
-
-### Footer controls block
-
-The `footer` block can yield a `cancel`, `confirm` or generic button type as `controls` block param. The `cancel` button is internally hooked up to close the dialog.
-The button text can be customized using the `text` attribute. The `cancel` button text defaults to `Cancel`. The `confirm` button text defaults to `Confirm`.
-The `confirm` button is a primary button that takes a `onConfirm` action handler to allow the consumer to perform actions on confirmation, eg saving a record, and it  closes the dialog by default. If however, you set the template attr `closeOnClick=false`, it will not close the dialog on confirm click. This allows you to hook into the confirm action and perform other tasks.
-The footer block also yields a `close` action that a consumer can call to correctly close the dialog after all tasks are completed.
-If you require another custom button, you can use the generic button type. To set an action handler on that button, set the attr `onClick`. All button text, size and priority can be overridden.
-
-### ember-perfectscroll effects
+### ember-perfectscroll effects (TODO effects based of perfectscroll events, but not from perfectscroll)
```

Comment:
I'd probably remove this section anyway - it's covered in the demo and is a detail generally hidden from most users.